go-monorepo/gotidy.sh

```diff
@@ -34,6 +34,8 @@ if [ "$1" = "-u" ]; then
 		exit 1
 		;;
 	esac
+elif [ "$1" = "--build" ]; then
```

Comment:
The if-else chain only handles one argument, so `-u --build` won't work as expected. Maybe make it so `-u` always implies `--build`?